express-post-json/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+
+const grades = [];
+let nextId = 1;
+
+const files = express.json();
+app.use(files);
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+
+  res.status(201).send(grades);
+  req.body.id = nextId;
+  grades.push(req.body);
+  nextId++;
```

Comment:
The client should only receive the new object (not an array) after it's been given the id.